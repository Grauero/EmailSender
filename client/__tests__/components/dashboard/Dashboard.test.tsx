import 'jest';
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Dashboard from '../../../src/components/dashboard/Dashboard';

test('matches snapshot', () => {
  expect(toJSON(shallow(<Dashboard />))).toMatchSnapshot();
});
