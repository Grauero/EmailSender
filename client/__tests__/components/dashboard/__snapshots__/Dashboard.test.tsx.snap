// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="content"
>
  <Connect(SurveyList) />
  <div
    className="fixed-action-btn"
  >
    <Link
      className="btn-floating btn-large red"
      replace={false}
      to="/surveys/new"
    >
      <i
        className="material-icons"
      >
        add
      </i>
    </Link>
  </div>
</div>
`;
